tests/Kernel/MediaRenderTest.php

```diff
@@ -59,45 +62,132 @@ protected function setUp() {
    * Tests that the Document media type is rendered with the correct ECL markup.
    */
   public function testDocumentMedia(): void {
-    $file = file_save_data(file_get_contents(drupal_get_path('module', 'oe_media') . '/tests/fixtures/sample.pdf'), 'public://test.pdf');
-    $file->setPermanent();
-    $file->save();
+    // Make image media translatable.
+    $setting = $this->container->get('entity_type.manager')->getStorage('language_content_settings')->create([
+      'langcode' => 'en',
+      'status' => TRUE,
+      'id' => 'media.document',
+      'target_entity_type_id' => 'media',
+      'target_bundle' => 'document',
+      'default_langcode' => 'site_default',
+      'language_alterable' => TRUE,
+    ]);
+    $setting->setThirdPartySetting('content_translation', 'enabled', TRUE);
+    $setting->save();
```

Comment:
I believe you could make this just in two lines:
`$this->container->get('content_translation.manager')->setEnabled('media', 'document', TRUE);
 $this->container->get('router.builder')->rebuild();`